src/components/welcome/MyAccounts/useSafesSearch.ts

```diff
@@ -0,0 +1,22 @@
+import { useMemo } from 'react'
+import Fuse from 'fuse.js'
+import type { MultiChainSafeItem } from './useAllSafesGrouped'
+import type { SafeItem } from './useAllSafes'
+
+const useSafesSearch = (safes: (SafeItem | MultiChainSafeItem)[], query: string) => {
+  const fuse = useMemo(
+    () =>
+      new Fuse(safes, {
+        keys: [{ name: 'name' }],
```

Comment:
Should we also allow searching by address? I quickly tested this and it only worked when typing from the start e.g. `0x7a...` but not if I type characters at the end of the address I want to find e.g. if my address ends in `86E2` and I type `86E` there is no result. Is this possible with fuse.js?

Reply:
Another idea to include searching by prefix e.g. if the address includes `sep:0x7a...` searching for "sep" would only show sepolia safes.

Reply:
Good idea! We decided against searching for addresses to keep things simple but it seems like it could be useful and it's not so much extra effort. And having a way to filter by chain would be quite useful I think. Let's add them in